.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,95 @@
+name: Bug Report
+description: Create a bug report to help us improve
+title: "[Bug]: "
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to fill out this bug report!
+  - type: markdown
+    attributes:
+      value: Please make sure to provide the information we ask for. This will allow us to help you better.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: A bug happened! 'foo' should not be there.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behaviour
+      description: A clear and concise description of what you expected to happen.
+      placeholder: I expected to see X or Y instead of 'foo'
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to reproduce
+      description: Please add link to a repo with a minimal reproduction. Or describe as detailed as possible how we can reproduce/verify the bug.
```

Would it be better to say repository instead of repo (there might be people unfamiliar with the abbreviation). I don't think `detailed` can be used as an adverb (no examples in https://www.oxfordlearnersdictionaries.com/definition/english/detailed?q=detailed).
```suggestion
      description: Please add a link to a repository with a minimal reproduction. Or accurately describe how we can reproduce the bug.
```